maltd-frontend/src/components/hoc/AuthenticationGuard.js

```diff
@@ -46,8 +50,15 @@ export default function AuthenticationGuard() {
       })
       .success(() => {
         keycloak.loadUserInfo().success();
-        localStorage.setItem("jwt", keycloak.token);
-        setAuthedKeycloak(keycloak);
+        if(accessRole != null && accessRole != "" && keycloak.tokenParsed.realm_access.roles.indexOf(accessRole) != -1){
```

Comment:
Should just be able to check for all falsey values using `if(accessRole)`
That will prevent something from happening if `accessRole` is null, undefined, empty string. Also, may be better to use `===` for equality checks as they’re safer in javascript due to type checking :)

So like;

`if (accessRole && keycloak.tokenParsed.realm_access.roles.indexOf(accessRole) !== -1)`

Reply:
Yes, I actually was about to submit the === fix, I wrote this in a hurry :)

Also will change to the better falsey check